src/main/java/seedu/address/model/person/UniquePersonList.java

```diff
@@ -103,6 +105,15 @@ public void setPersons(List<? extends ReadOnlyPerson> persons) throws DuplicateP
         return FXCollections.unmodifiableObservableList(mappedList);
     }
 
+    /**
+     * Sorts person name in alphabetical order
+     */
+    public void sort() {
+        Comparator<Person> comparator = (p1, p2) -> (p1.getName().toString().compareTo(p2.getName().toString()));
```

Comment:
Name class already got an overridden `toString()` method, so no need to have `toString()` here.

Reply:
Without toString(), compareTo cannot be used.

Reply:
ok my bad.